aip_x1_launch/launch/topic_state_monitor.launch.py

```diff
@@ -155,7 +155,7 @@ def generate_launch_description():
         name="topic_state_monitor_rough_no_ground",
         parameters=[
             {
-                "topic": "/perception/object_segmentation/ground_segmentation/no_ground/pointcloud",
+                "topic": "/perception/obstacle_segmentation/ground_segmentation/no_ground/pointcloud",
```

Comment:
```suggestion
                "topic": "/perception/obstacle_segmentation/pointcloud",
```

Reply:
May I fix these points (remove `no_ground` word) in the other PRs ?

Reply:
okay :+1: